commit 96a7037cd8573cf065aa6b12baca68696f96d9ca
Author: Sergei Trofimovich <siarheit@google.com>
Date:   Mon Jul 26 22:51:18 2021 +0100

    texi2pod.pl: add no-op --no-split option support [PR28144]
    
    Change 2faf902da ("generate single html manual page by default")
    added use of --no-split option to makeinfo. binutils reuses
    makeinfo options for texi2pod.pl wrapper. Unsupported option
    led to silent manpage truncation.
    
    The change adds no-op option support.
    
    etc/
    
            * texi2pod.pl: Handle no-op --no-split option.

diff --git a/etc/texi2pod.pl b/etc/texi2pod.pl
index 11f70d156b..dcf2b43764 100644
--- a/etc/texi2pod.pl
+++ b/etc/texi2pod.pl
@@ -59,6 +59,8 @@ while ($_ = shift) {
 	    $flag = shift;
 	}
         push (@ipath, $flag);
+    } elsif (/^--no-split$/) {
+	# ignore option for makeinfo compatibility
     } elsif (/^-/) {
 	usage();
     } else {
